samples/subsys/ipc/ipm_psoc6/README.rst

```diff
@@ -0,0 +1,74 @@
+.. _ipm-psoc6-sample:
+
+Sample mailbox application
+##########################
+
+Overview
+********
+
+The :ref:`cy8ckit_062_wifi_bt` board has two core processors (Cortex-M4
+and Cortex-M0+). This sample application uses a mailbox to send messages
+from Cortex-M4 processor core (client) to Cortex-M0+ (server).
+
+Requirements
+************
+
+- :ref:`cy8ckit_062_wifi_bt` board
+
+Building and Running
+********************
+
+#. Build the project
+
+.. code-block:: console
+
+   $ cd samples/subsys/ipc/ipm_psoc6
+   $ mkdir build
+   $ cd build
+   $ cmake -G"Eclipse CDT4 - Ninja" ..
+   $ ninja
+
+#. Switch the DevKit into CMSIS-DAP mode using SW3 (LED2 should blink) and
+flash the board:
+
+.. code-block:: console
```

Comment:
as above, indent the code block (and body) by three spaces.  (The ``..`` should align with the ``S`` in ``Switch``.)